src/main/resources/sentry.properties

```diff
@@ -0,0 +1 @@
+dsn=
```

Comment:
does this work?

Reply:
Yes, it's the only thing that does.

Reply:
I was planning to move all of the default config from SentryServletContainerInitializer to this file, but given how much of a pain in the neck this was I didn't want to do more than what was required. So I'll add a ticket to handle that and figuring out a way to get the SentryServletContainerInitializer to work in all environments or replace it with custom init method in BootStrap.groovy.